data/json/items/tools.json

```diff
@@ -1968,6 +1984,28 @@
       "not_ready_msg": "The eggs are not done yet."
     }
   },
+  {
+    "id": "milk_cream_rising",
+    "type": "GENERIC",
+    "category": "food",
+    "name": "milk cream jar",
+    "description": "This jar contains raw milk separating into cream.  You can seal up the jar once the process is completed.",
+    "weight": "52 g",
+    "volume": "3000 ml",
+    "price": 10,
+    "bashing": 8,
+    "material": [ "glass", "dairy" ],
+    "symbol": "%",
+    "color": "white",
+    "use_action": {
+      "target": "milk_cream_jar",
```

Comment:
This guy here, I don't see it elsewhere, am I blind?

Reply:
I was having troubles with byproducts and transforms so I pulled "all" (missed one) of the milk cream items.  I was able to fix the byproducts while away from the internet and can load playable games but the transform items are spitting out a bunch of errors still.  Also the actual process of separating cream from milk takes basically the entire lifespan of CDDA milk.  So I was trying to create a starting point for crafting butter then going back and figuring out the actual lifespan of raw milk with data points for why it should be changed in game.  But the cream jar should work as basically you can get cream from storing raw milk in a jar and letting the cream rise.  I figured for fun over realism that it would be best to model after saurkraut or some other activated jar since coming back to a craft exactly 12-18 hours after you make it would be too complicated.

Reply:
This is part of where I stopped when working on butter: I'd wanted to have have rotten milk turn into 'sour' milk when it turned, which would then be used to make buttermilk and cream. I was unable to get a working `rot_transform` thing going, so I ended up shelving it.

Reply:
I am but I need to spend the next week or two finishing my last class for this semester.  So fixing the mass of errors in my transforms is probably going to take a backburner for a bit. If you have a second please check out where I've added the delay transforms I've added back in.